test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/AllowListVerificationProviderTests.cs

```diff
@@ -1213,6 +1013,37 @@ public AllowListVerificationProviderTests(SigningTestFixture fixture)
                 repoAllowListEntries: repoAllowList,
                 clientAllowListEntries: clientAllowList);
         }
+
+        public static IEnumerable<object[]> EmptyNullAndRequiredListCombinations()
+        {
+            // NoAllowUntrusted_RepoListNotRequire_ClientListRequired_NoClientAllowList_NoRepoAllowList_Error
+            yield return new object[]
+            { GetSettings(false, true, false, null, null), false, 1, 1, new object[]{  new object[]{ NuGetLogCode.NU3034, _noClientAllowList } } };
```

Comment:
This is great, but it would it would greatly aid readability if you used named parameters or something like a table header.

Basically, as I look this over to understand the set of tests, it's difficult to understand each test; I have to go back to `GetSettings(...)` to see what each parameter is so I know what `true` or 0 means.

Reply:
I agree, I didn't want to blow up each line though, that's why I added the comment on top, which is also in the same order as the params so once you realize that it will be easy to just read the comments and figure out what the different scenarios are